dataactvalidator/app.py

```diff
@@ -102,7 +103,7 @@ def run_app():
 
             except ResponseException as e:
                 # Handle exceptions explicitly raised during validation.
-                logger.error(str(e))
+                logger.error(traceback.print_exc())
```

Comment:
Are we sure this works? How does it know what it's printing?

Reply:
It prints the exception that was caught in the current thread. I've used it pretty extensively, and also, I had the Inbound API user who was running broker locally make this change in his local file in order to figure out what his issue was.